packs/rabbitmq/sensors/queues_sensor.py

```diff
@@ -0,0 +1,45 @@
+from st2reactor.sensor.base import PollingSensor
+import pika
+from pika.credentials import PlainCredentials
+
+
+class RabbitMQSensor(PollingSensor):
```

Comment:
For the next time, we should be more explicit with the class name. I am assuming we'll write more such sensors. RabbitMQSensor is way too generic :P.